src/store/menu/__tests__/menu.reducers.test.ts

```diff
@@ -0,0 +1,20 @@
+import { menuReducer } from 'store/menu/reducers/menu.reducer';
+import { initialState } from '../hardcoded-data';
```

Comment:
nit: There are terms like stubs and mocks. You can use them to name your files to emphasize that used data is a fake.

Reply:
done